src/AspNet.Security.OAuth.Shopify/ShopifyAuthenticationOptions.cs

```diff
@@ -0,0 +1,69 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System.Security.Claims;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.OAuth;
+using Microsoft.AspNetCore.Http;
+
+namespace AspNet.Security.OAuth.Shopify
+{
+    /// <inheritdoc />
+    public class ShopifyAuthenticationOptions : OAuthOptions
+    {
+        /// <summary>
+        /// Claim valueType used for boolean claims.
+        /// </summary>
+        public const string XmlSchemaBoolean = "http://www.w3.org/2001/XMLSchema#boolean";
+
+        /// <summary>
+        /// Claim valueType used for dateTime claims (iso format)
+        /// </summary>
+        public const string XmlSchemaDateTime = "http://www.w3.org/2001/XMLSchema#dateTime";
```

Comment:
This is a built-in framework constant, we shouldn't need to redefine it: https://github.com/dotnet/corefx/blob/e12bfba58e3581dc475e804688bef1c66d70226c/src/System.Security.Claims/src/System/Security/Claims/ClaimValueTypes.cs#L18